kasa/device.py

```diff
@@ -547,10 +547,7 @@ def _get_replacing_attr(
     def __getattr__(self, name: str) -> Any:
         # is_device_type
         if dep_device_type_attr := self._deprecated_device_type_attributes.get(name):
-            module = dep_device_type_attr[0]
-            msg = f"{name} is deprecated"
-            if module:
-                msg += f", use: {module} in device.modules instead"
```

Comment:
This is used only for `is_x` properties, so no need to print out "module"/None.

Reply:
ohh. hmm. So I noticed that it was recommending to use `Led` instead of `is_plug`, which felt very wrong. I'd guess recommending to use `DeviceType` is more relevant for most users anyway.

Reply:
I think the issue was device_type is very singular and we didn't really want consumers to test for it at all, instead if the module is there you can test those features/components instead of 1:1 mapping to device_types.

Reply:
As it's all deprecated and obsolete, it doesn't really matter that much, IMO.